reference/3.0/Microsoft.PowerShell.Core/About/about_Execution_Policies.md

```diff
@@ -310,42 +322,43 @@ For more information, see [about_Group_Policy_Settings](about_Group_Policy_Setti
 When determining the effective execution policy for a session, PowerShell
 evaluates the execution policies in the following precedence order:
 
-- Group Policy: Computer Configuration
-- Group Policy: User Configuration
+- Group Policy: MachinePolicy
+- Group Policy: UserPolicy
 - Execution Policy: Process (or `powershell.exe -ExecutionPolicy`)
 - Execution Policy: CurrentUser
 - Execution Policy: LocalMachine
 
 ## Manage Signed and Unsigned Scripts
 
-If your PowerShell execution policy is RemoteSigned, PowerShell will not run
-unsigned scripts that are downloaded from the Internet which includes e-mail
-and instant messaging programs.
+If your PowerShell execution policy is **RemoteSigned**, PowerShell will not
+run unsigned scripts that are downloaded from the Internet which includes
+e-mail and instant messaging programs.
```

Comment:
Current AP style for 'e-mail' is 'email'.

Reply:
Resolved, thanks for the feedback.